show_all.rego

```diff
@@ -1,5 +1,12 @@
-package policy
+# METADATA
+# title: Show All
```

Comment:
The title is "show all", but the policy blocks all issues.

Reply:
Updated and changed file name to use the "block" language